lib/hooks/useGetUserDetailsFromDatabase.ts

```diff
@@ -19,18 +20,22 @@ export const useSearchByUsernameOrAddress = (searchQuery: string) => {
   const [error, setError] = useState<Error | null>(null);
   const [isLoading, setIsLoading] = useState<boolean>(false);
   const { supabase } = useSupabase();
-
+  const provider = new ethers.CloudflareProvider();
```

Comment:
We Can use this 
https://github.com/Seroxdesign/MetaLinks/blob/main/utils/ensUtils.ts